appcenter-auth/Auth.js

```diff
@@ -6,18 +6,45 @@ const ReactNative = require('react-native');
 const { AppCenterReactNativeAuth } = ReactNative.NativeModules;
 
 const Auth = {
-    isEnabled() {
-        return AppCenterReactNativeAuth.isEnabled();
-    },
-    setEnabled(enabled) {
-        return AppCenterReactNativeAuth.setEnabled(enabled);
-    },
-    signIn() {
-        return AppCenterReactNativeAuth.signIn();
-    },
-    signOut() {
-        AppCenterReactNativeAuth.signOut();
-    }
+    isEnabled,
+    setEnabled,
+    signIn,
+    signOut
 };
 
+/**
+ * Check whether Auth service is enabled or not.
+ *
+ * @return Promise with the result being true if enabled, false otherwise.
```

Comment:
I believe the terminology for promises is 'fulfilled', so perhaps something like this?

```suggestion
 * @return A promise fulfilled with true if enabled, false otherwise.
```

Reply:
Updated, thanks @jwallra!